Description: Fix FTBFS due to unexisting and useless include. (Closes: #637896)
Author: Regis Boudin <regis@debian.org>
Last-Update: 2011-10-03
Index: libofa-0.9.3/examples/protocol.cpp
===================================================================
--- libofa-0.9.3.orig/examples/protocol.cpp
+++ libofa-0.9.3/examples/protocol.cpp
@@ -13,7 +13,6 @@
 #include <map>
 #include <expat.h>
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 
 using namespace std;
